lib/vmpooler/pool_manager.rb

```diff
@@ -26,6 +28,9 @@ def initialize(config, logger, redis, metrics)
       @reconfigure_pool = {}
 
       @vm_mutex = {}
+
+      # load specified providers from config file
+      load_used_providers
```

Comment:
All the providers referenced in the config are now loaded here.  Previously these were loaded during the loading of the vmpooler file.